src/DssSpell.sol

```diff
@@ -42,38 +42,239 @@ contract DssSpellAction is DssAction {
     // A table of rates can be found at
     //    https://ipfs.io/ipfs/QmefQMseb3AiTapiAKKexdKHig8wroKuZbmLtPLv4u2YwW
     //
-    uint256 constant ZERO_FIVE_PCT_RATE = 1000000000158153903837946257;
 
-    uint256 constant MILLION  = 10**6;
+    // --- Rates ---
+    uint256 constant FOUR_PCT_RATE           = 1000000001243680656318820312;
+    uint256 constant SEVEN_PCT_RATE          = 1000000002145441671308778766;
+
+    // --- Math ---
+    uint256 constant MILLION                 = 10 ** 6;
+    uint256 constant RAY                     = 10 ** 27;
+
+    // --- WBTC-B ---
+    address constant MCD_JOIN_WBTC_B         = 0xfA8c996e158B80D77FbD0082BB437556A65B96E0;
+    address constant MCD_CLIP_WBTC_B         = 0xe30663C6f83A06eDeE6273d72274AE24f1084a22;
+    address constant MCD_CLIP_CALC_WBTC_B    = 0xeb911E99D7ADD1350DC39d84D60835BA9B287D96;
+
+    // --- Offboarding: Current Liquidation Ratio ---
+    uint256 constant CURRENT_AAVE_MAT        =  165 * RAY / 100;
+    uint256 constant CURRENT_BAL_MAT         =  165 * RAY / 100;
+    uint256 constant CURRENT_COMP_MAT        =  165 * RAY / 100;
+
+    // --- Offboarding: Target Liquidation Ratio ---
+    uint256 constant TARGET_AAVE_MAT         = 2100 * RAY / 100;
+    uint256 constant TARGET_BAL_MAT          = 2300 * RAY / 100;
+    uint256 constant TARGET_COMP_MAT         = 2000 * RAY / 100;
+
+    // --- OLD LERP FAB ---
+    address constant OLD_LERP_FAB            = 0x00B416da876fe42dd02813da435Cc030F0d72434;
```

Comment:
If in mainnet the new lerp fab was already being used for the lerps that we want to replace then use that one and avoid setting this double thing.

Reply:
This comment doesn't apply as there was a new one, it was a problem of duplicated key in the addresses file.